http-server/src/main/java/io/micronaut/http/server/HttpServerConfiguration.java

```diff
@@ -1045,4 +1051,116 @@ public void setHeader(boolean header) {
             this.header = header;
         }
     }
+
+    /**
+     * Allows configuration of properties for the {@link io.micronaut.http.server.netty.body.AbstractFileBodyWriter}.
+     *
+     * @author James Kleeh
+     * @author graemerocher
+     * @since 3.1.0
+     */
+    @ConfigurationProperties("responses.file")
+    public static class FileTypeHandlerConfiguration {
```

Comment:
Is is a new file?

Reply:
I'm just moving the `micronaut.server.netty.responses.file` config to `micronaut.server.responses.file`, not sure if this is the best way